woocommerce/storefront
# check if Storefront_NUX_Admin_Inbox_Messages_Customize is defined bef…

Gravity forms settings page seems to load certain hook twice, most precisely, `admin_notices` action, this caused `class-storefront-nux-admin-inbox-messages-customize.php` to be required twice and `Storefront_NUX_Admin_Inbox_Messages_Customize` to be defined twice.

Other possible solutions include:
- importing `class-storefront-nux-admin-inbox-messages-customize.php` outside the hook.
- Understanding why Gravity forms is triggering this action twice and fixing it.

I opted for checking if the class is already defined.

### Testing instructions
- Download a testing copy of gravity forms premium from this private slack thread p1621431332030900-slack-C7U3Y3VMY
- Go to Gravity Forms settings page
- It should load fine

## Changelog entry
> Fix compatibility problem with Gravity forms premium
